nano/node/network.cpp

```diff
@@ -236,6 +236,28 @@ void nano::network::flood_message (nano::message const & message_a, bool const i
 	}
 }
 
+void nano::network::flood_vote (std::shared_ptr<nano::vote> vote_a, float scale)
+{
+	auto list (list_fanout (scale));
+	for (auto i (list.begin ()), n (list.end ()); i != n; ++i)
+	{
+		if (!node.rep_crawler.is_pr (**i))
+		{
+			(*i)->send (nano::confirm_ack (vote_a));
+		}
+	}
+}
+
+void nano::network::flood_vote_pr (std::shared_ptr<nano::vote> vote_a)
+{
+	nano::confirm_ack message (vote_a);
+	auto list = node.rep_crawler.principal_representatives ();
+	for (auto const & i: list)
```

Comment:
I guess for consistency usually we have extra space between `i` and  `:`

Reply:
Yes